### ISSUE TITLE
bug/ Coordinates returned by the `by_title` chunking strategy only contains the title

### ISSUE THREAD

**Issue by GilbertHLam** [Unstructured-IO/unstructured#1698]
**Describe the bug**
When using the `by_title` chunking strategy, it looks like the coordinates returned for each section only contains the title or first  element in the section. This is not an issue when not using the `by_title` strategy

**To Reproduce**
We are using the Unstructure API with these options 

```
      strategy: "auto",
      output_format: "json",
      ocr_languages: "eng",
      encoding: "utf-8",
      coordinates: true,
      include_page_breaks: true,
      multipage_sections: false,
      chunking_strategy: "by_title",
      combine_under_n_chars: 250,
      new_after_n_chars: 1500
```

**Expected behavior**
I would expect the coordinates to contain all the text within the section

**Screenshots**

**Environment Info**
Please run `python scripts/collect_env.py` and paste the output here. 
This will help us understand more about the environment in which the bug occurred.

**Additional context**

**Comment by scanny** [comment]
Hi @GilbertHLam, this behavior was addressed a couple weeks ago in this PR: #1947.

We considered forming a "consolidated" bounding box from the extremes of those in the elements represented in the chunk but rejected that idea for the moment because elements can span page boundaries and coordinates are inherently page-based. So the new behavior is that coordinates are dropped during chunking.

Let us know if you think we missed something there, but closing this for now as resolved :)

**Comment by scanny** [comment]
Yes, that thought did occur to me and I think it's worth suggesting if you're willing to open a new issue for it.

It might be entitled "Include consolidated bounding box in chunk metadata when chunking by page".

There could be two cases where this would be possible, when the chunking strategy is "by_page" and also when it is "by_title" with the argument `multipage_sections = False`.
